src/master/x11.rs

```diff
@@ -1,68 +1,48 @@
+use crate::{CallbackResult, ClipboardHandler, Master};
 use std::io;
-use crate::{ClipboardHandler, CallbackResult, Master};
-
-use x11_clipboard::xcb;
 
 impl<H: ClipboardHandler> Master<H> {
     ///Starts Master by waiting for any change
     pub fn run(&mut self) -> io::Result<()> {
-        let mut result = Ok(());
-        let clipboard = x11_clipboard::Clipboard::new().unwrap();
-
-        let xfixes = match xcb::query_extension(&clipboard.getter.connection, "XFIXES").get_reply() {
-            Ok(xfixes) => xfixes,
-            Err(error) => return Err(io::Error::new(io::ErrorKind::Other, error)),
-        };
-        assert!(xfixes.present());
-        xcb::xfixes::query_version(&clipboard.getter.connection, 5, 0);
+        let clipboard = x11_clipboard::Clipboard::new();
```

Comment:
Can you please check you can perform clipboard operations from within handler?
(write string or read string)

Reply:
Sorry, I am not sure what do you mean to perform operation from within handler.

Reply:
What I mean is that can you create new `x11_clipboard::Clipboard` and perform actual load without losing content.

Using `load_wait` will load content and clear it from selection.